components/ProfileSubmissions.tsx

```diff
@@ -1,5 +1,8 @@
 import React from 'react'
 import '../scss/profileSubmissions.scss'
+import '../scss/profileLessons.scss'
```

Comment:
I don't see this `scss` file. Do you want to put the `css` into `profileSubmissions.scss` instead?